lib/schema/organization.ts

```diff
@@ -0,0 +1,14 @@
+import * as Joi from 'joi';
+import { Regexes } from '../regexes';
+
+export enum GroupSize {
+  OneHundredOrGreater = '100 or More Individual Eligible Clinicians',
+  TwentyFiveNinetyNine = '25 - 99 Individual Eligible Clinicians',
+  TwoTwentyFour = '2 - 24 Individual Eligible Clinicians'
+}
+
+export const OrganizationSchema = Joi.object().keys({
+  name: Joi.string().max(128).regex(Regexes.containsNonWhitespace, 'ContainsNonWhitespace').required(),
+  nickname: Joi.string().max(128).regex(Regexes.containsNonWhitespace, 'ContainsNonWhitespace').allow(null),
+  groupSize: Joi.any().valid(GroupSize.OneHundredOrGreater, GroupSize.TwentyFiveNinetyNine, GroupSize.TwoTwentyFour)
```

Comment:
Can we make `groupSize` a string instead of any?

Reply:
I couldn't find anything like .valid() for a Joi.string().  I could switch it to a regex, but I like the valid() syntax better.

Reply:
https://github.com/hapijs/joi/blob/master/API.md#anyvalidvalue---aliases-only-equal

Reply:
Nevermind, it looks like string() inherits from any() so I get all the same functions